blob/src/test/java/com/bazaarvoice/emodb/blob/core/DefaultBlobStoreTest.java

```diff
@@ -20,29 +20,28 @@
 import java.util.Map;
 import java.util.UUID;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
```

Comment:
Curious how did it work. I don't see junit dependency in blob module.

Reply:
`junit` is transitive dependency of `testng`